browsermt/marian-dev
# Compile libmariandecoder to wasm

### Description
This PR adds WASM builds of marian-decoder
It is related to issues: #5

List of changes:
- 
- ...
- ...

Added dependencies: Docker

### How to test

1. Install Docker if not already installed
2. Checkout the compile-wasm branch
3. See the [README](https://github.com/browsermt/marian-dev/blob/compile-wasm/docker/README.md#wasm)

Expected results during draft stage of this PR: The compilation fails on Windows

### Checklist
